salt/master.py

```diff
@@ -1452,7 +1452,7 @@ def _file_recv(self, load):
             if load['loc']:
                 fp_.seek(load['loc'])
 
-            fp_.write(load['data'])
+            fp_.write(load['data'].encode('utf8'))
```

Comment:
This needs to be changed to `salt.utils.stringutils.to_bytes(load['data'])`.

Reply:
Thanks! Changing it now.